Class diagram

felipeaugustosm opened the pull request:
Primeira versão já com as alterações feitas, após a reunião do dia 27/03/2018

felipeaugustosm committed:
Primeira Versão com Alterações

Primeira versão já com as alterações feitas, após a reunião do dia 27/03/2018

ChristyanS committed:
Adding Style to class diagram

ChristyanS committed:
Adding Style to class diagram

ChristyanS reviewed (changes requested):
My analysis about class diagram:
In my opinion, the class 'Usuario' doesn't need s 'Endereço'
Maybe the multiplicity of 'Cartao de Credito' can be change for N N
The class 'Local' don't needed of 'tipo residencia'
WishList to 'Evento' should be N N
'Evento' e 'Contato' too should be N N
The class 'Contatos' should be called 'Contato'
I think that don't need an association between 'Contatos' and 'Eventos' because 'Organizador' already has 'Contatos' 
Maybe Eventos's email can be removed 'cause 'Organizador' already email
In the classe 'Ingressos' is necessary the fields 'Nome ','Sobrenome','Celular','Email' because the ticked is nominal.
The Class 'Promoção' should't have specials caracters
Is necessary a field to 'avaliações'

felipeaugustosm committed:
Alterações baseado nas observações do Christian

Foram feitas alterações no diagrama, baseado nas observações feitas pelo Christian.

felipeaugustosm committed:
Branch inicial
